app/views/taxon_framework_relationships/_show_taxon_framework_relationship_container.html.haml

```diff
@@ -0,0 +1,24 @@
+:ruby
+  taxon_framework_relationship ||= show_taxon_framework_relationship_container[:taxon_framework_relationship] ||= @taxon_framework_relationship
+  taxon ||= show_taxon_framework_relationship_container[:taxon] ||= @taxon
+  upstream_taxon_framework ||= show_taxon_framework_relationship_container[:taxon_framework] || @upstream_taxon_framework || @taxon_framework
+.panel.panel-default
+  .panel-heading
+    - if is_admin? && upstream_taxon_framework.source && !taxon_framework_relationship
+      .pull-right
+        .admin-box
+          = link_to "Add relationship", new_taxon_framework_relationship_path( { taxon_id: taxon.id, taxon_framework_id: upstream_taxon_framework.id } )
+    %h4
+      = t( :covered_by_a_taxon_framework_html, href: link_to( t( :covered_by_a_taxon_framework_href, taxon_framework_rank: t( "ranks.#{ upstream_taxon_framework.taxon.rank }" ).capitalize, taxon_framework_name: upstream_taxon_framework.taxon.name ), taxonomy_details_for_taxon_path( upstream_taxon_framework.taxon ) ) )
```

Comment:
This is confusing. `href` is a full link, not an href attribute. Better to use something like `link`. Reading this, I thought for sure it was a bug.

Reply:
fixed